model/src/contract.rs

```diff
@@ -286,13 +286,43 @@ where
     }
 }
 
+/// A return status code for `deploy` and `dispatch` calls back to the SRML contracts module.
+#[derive(Debug, Copy, Clone, PartialEq, Eq)]
+pub struct RetCode(u32);
```

Comment:
Why not make this a `u8` to enforce the max `255`? We're calling `to_u32()` in the lang layer anyway.

Reply:
good point actually!